### PR TITLE
[jupyter_notebook] add guide for Jupyter Notebook

### LINKED ISSUE #684: Guide for Jupyter Notebook
I have just successfully deployed jupyter to my Uberspace. It wasn't super difficult but there were a few steps involved and I could write up a short guide on how to do it.

I have seen that there is interest in JupyterHub (#503). I have tried that as well, but it does not seem to work (as of yet) as JupyterHub heavily relies on having root access (sad). However the normal jupyter installation would already allow you to access your notebooks from the outside, but it'd be restricted to single-user (i.e. to share you have to give other people your password)

### PR DESCRIPTION
**Pull request by EV21**
closes #684 

- [x] Test this guide on a fresh U7. - done

### CODE REVIEW

File: source/guide_jupyter-notebook.rst [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+.. code-block:: console
+
+ [isabell@stardust ~]$ pip3 install --user --upgrade pip
+ Collecting pip
+  Using cached https://files.pythonhosted.org/packages/43/84/23ed6a1796480a6f1a2d38f2802901d078266bda38388954d01d3f2e821d/pip-20.1.1-py2.py3-none-any.whl
+ Installing collected packages: pip
+ Successfully installed pip-20.1.1
+ [isabell@stardust ~]$
+
+.. note:: If you are using :manual_anchor:`pip <lang-python.html#pip>` with the default configured Python 2.7.5 you may get a syntax error while the installation of jupyter. This guide uses pip3 so you do not need to change it.
```
Comment (FriedrichFroebel) [inline source/guide_jupyter-notebook.rst:47]: "while installing jupyter"

File: source/guide_jupyter-notebook.rst [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+.. code-block:: ini
+ :emphasize-lines: 4,5
+
+ c.NotebookApp.allow_password_change = False
+ c.NotebookApp.ip = '0.0.0.0'
+ c.NotebookApp.open_browser = False
+ c.NotebookApp.port = 8888
+ c.ContentsManager.root_dir = '/home/isabell/'
+
+.. note:: The parameter ``c.ContentsManager.root_dir`` sets the root of the Jupyter file manger. So you could browse your hole user directory with this setting.
```
Comment (FriedrichFroebel) [inline source/guide_jupyter-notebook.rst:90]: "whole" instead of "hole"

File: source/guide_jupyter-notebook.rst [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+Create ``~/etc/services.d/jupyter.ini`` with the following content:
+
+.. code-block:: ini
+
+ [program:jupyter]
+ command=jupyter notebook
+ autostart=true
+ autorestart=true
+
+After creating the configuration, tell :manual:`supervisord <daemons-supervisord>` to refresh its configuration and start the service:
```
Comment (FriedrichFroebel) [inline source/guide_jupyter-notebook.rst:116]: You should probably use the `supervisord.rst` snippet here.

File: source/guide_jupyter-notebook.rst [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+ [isabell@stardust ~]$ supervisorctl status
+ jupyter                            RUNNING   pid 26020, uptime 0:01:23
+ [isabell@stardust ~]$
+
+Setup web backend
+-----------------
+
+To make the application accessible from the outside, configure a :manual:`web-backend <web-backends>`.
+If you have your own :manual:`domain <web-domains>` you can also set a domain specific web beckend configuration.
+This guide overrides the default setting, so the default html folder is not accessable.
```
Comment (FriedrichFroebel) [inline source/guide_jupyter-notebook.rst:133]: "accessible"

File: source/guide_jupyter-notebook.rst (hunk 3 above)
Comment (EV21) [reply source/guide_jupyter-notebook.rst:116]: I didn't know this is possible. I also do this for the web backend.

**Comment by luto** [comment]
works great! thank you :)

**Pull request merged**